pipelines/smartseq2_single_sample/SS2SingleSample.wdl

```diff
@@ -0,0 +1,136 @@
+##### Naming Rules #####
+## ** Files should be named in CamelCase starting with a capital
```

Comment:
What are the capitalization rules when using an acronym (like SS, WGS, VCF, etc.)?

Reply:
good question and honestly I don't have a good answer of what the rules should be other than consistent 😞.  Something to bring up with the team for sure

Reply:
I vote for leaving them capitalized, but in cases of jargon acronyms, remove them and spell out the word if possible. (e.g. SS -> SmartSeq, WGS -> WholeGenomeSequencing). 

Conversely, things like BAM, VCF, FASTQ, that all have standards, can be left in.